apps/voting/app/src/components/VoteActions.js

```diff
@@ -0,0 +1,271 @@
+  const handleChangeVote = useCallback(() => setChangeVote(true), [])
+
+  const { connectedAccountVote, data } = vote
+  const { snapshotBlock, startDate, open } = data
+  const { canUserVote, canExecute, userBalance } = useExtendedVoteData(vote)
+  const hasVoted = [VOTE_YEA, VOTE_NAY].includes(connectedAccountVote)
+
+  if (!open) {
+    return (
+      <React.Fragment>
+        {canExecute && isVoteAction(vote) && (
+          <React.Fragment>
+            <Button mode="strong" onClick={onExecute} wide>
+              Enact this vote
+            </Button>
+            <Info>
+              The voting period is closed and the vote has passed.{' '}
+              <span css="font-weight: bold;">Anyone</span> can now enact this
+              vote to execute its action.
+            </Info>
+          </React.Fragment>
+        )}
+        {hasVoted && (
+          <div
+            css={`
+              border-radius: ${RADIUS}px;
+              background: ${theme.background};
+              padding: ${3.5 * GU}px ${10 * GU}px;
+              display: grid;
+              grid-template-columns: auto 1fr;
+              grid-gap: ${2 * GU}px;
+              align-items: center;
+            `}
+          >
+            <div>
+              <div
+                css={`
+                  border: 2px solid ${theme.accent};
+                  border-radius: 50%;
+                  width: 60px;
+                  height: 60px;
+                  display: flex;
+                  align-items: center;
+                  justify-content: center;
+                  color: ${theme.accent};
+                `}
+              >
+                <IconCheck />
+              </div>
+            </div>
+            <div>
+              <div
+                css={`
+                  ${textStyle('body1')}
+                  margin-bottom: ${0.5 * GU}px;
+                `}
+              >
+                Vote success
+              </div>
+              <div
+                css={`
+                  ${textStyle('body2')}
+                  color: ${theme.surfaceContentSecondary};
+                `}
+              >
+                You voted{' '}
+                <span
+                  css={`
+                    color: ${theme.surfaceContent};
+                    font-weight: 600;
+                    text-transform: uppercase;
+                  `}
+                >
+                  {connectedAccountVote === VOTE_YEA ? 'yes' : 'no'}
+                </span>{' '}
+                with{' '}
+                <span
+                  css={`
+                    color: ${theme.surfaceContent};
+                    font-weight: 600;
+                  `}
+                >
+                  {userBalance === -1 ? '…' : userBalance} {tokenSymbol}
+                </span>
+                .
+              </div>
+            </div>
+          </div>
+        )}
+      </React.Fragment>
+    )
+  }
+
+  if (canUserVote && hasVoted && !changeVote) {
```

Comment:
Did we decide to not show the "vote success" bit when you can change your vote ([design](https://www.figma.com/file/O09HJRCFDwDTni4VE3tiRvwI/Voting?node-id=1266%3A0))?

This in particular seems to have deviated from the design a bit. It currently looks like:

<img width="858" alt="Screen Shot 2019-08-12 at 1 16 40 AM" src="https://user-images.githubusercontent.com/4166642/62840679-f0a96a00-bc9e-11e9-9d8f-622861b86cd6.png">

Reply:
No, the feedback module (vote success) should be display above the button and info box. I added all the possible vote status to the Figma file. This screen should look like this: 

<img width="1534" alt="VOTING - PASSED" src="https://user-images.githubusercontent.com/7205369/62857357-b28c6480-bcf7-11e9-8eca-88d8175c80fc.png">